server/blob/blob-cassandra/src/main/java/org/apache/james/blob/cassandra/cache/CassandraBlobStoreCache.java

```diff
@@ -88,7 +100,11 @@
                     .setConsistencyLevel(ONE)
                     .setReadTimeoutMillis(readTimeOutFromDataBase)
             )
-            .map(this::toByteArray);
+            .map(this::toByteArray)
+            .map(bytes -> {
+                metricRetrieveHitCount.increment();
+                return metricFactory.runPublishingTimerMetric(BLOBSTORE_CACHED_LATENCY_METRIC_NAME, () -> bytes);
```

Comment:
You need to include the Cassandra query in time measurement.

Reply:
I realize it right after i pushed the commit.